docs/application_profile.md

```diff
@@ -81,7 +81,7 @@ The DLME Prototype Application Profile a subset of the [Europeana Data Model](ht
 | agg_data_provider  | `edm:dataProvider`  | M           | 1...1       | literal     | The organization or entity that supplies data through a provider. | |
 | agg_dc_rights      | `dc:rights`         |             | 0...n       | literal     | Information about rights held in and over the aggregation (both the Web Resource and the CHO). | Ideally this should be applied to the `edm:WebResource` or the `edm:ProvidedCHO`. |
 | agg_edm_rights     | `edm:rights`        | R           | 1...n       | reference   | Information about copyright, usage and access rights of the digital objects that represent the source cultural heritage object described in the data. Requires normalization. | The URI for a rights statement that applies to the object. |
-| agg_has_view       | `edm:hasView`       |             | 0...n       | JSON literal | URL reference for the digital object in an undetermined context (is the parent of `agg_is_shown_at`, `agg_is_shown_by`, and `agg_preview`) | The URL of a web resource which is a digital representation of the CHO. |
+| agg_has_view       | `edm:hasView`       |             | 0...n       | `edm:WebResource` | URL reference for the digital object in an undetermined context (is the parent of `agg_is_shown_at`, `agg_is_shown_by`, and `agg_preview`) | The URL of a web resource which is a digital representation of the CHO. |
```

Comment:
We should probably say it's a JSON literal representation of an `edm:WebResource`